controllers/rcj_soccer_player_b1/rcj_soccer_player_b1.py

```diff
@@ -1,9 +1,10 @@
-"""rcj_soccer_player controller."""
+"""rcj_soccer_player controller - ROBOT B1"""
```

Comment:
@Adman I really believe these should be comments -- if I am reading it correctly, this first line would normally be a [docstring for a module](https://stackoverflow.com/questions/708649/python-comments-vs-strings) (which our script most probably is not)